### PR TITLE
GG-21492 Try to calculate a fit measure for the size fields.

### PR DESCRIPTION
**Pull request by vsisko**

### CODE REVIEW

File: modules/web-console/frontend/app/components/form-field/components/form-field-size/controller.ts [hunk 1]
```diff
@@ -76,7 +76,16 @@ export default class PCFormFieldSizeController<T> implements IInputErrorNotifier
         if (!this.min) this.min = 0;
         if (!this.sizesMenu) this.setDefaultSizeType();
         this.$element.addClass('form-field');
-        this.ngModel.$render = () => this.assignValue(this.ngModel.$viewValue);
+        this.ngModel.$render = () => {
```
Comment (akuznetsov-gridgain) [inline modules/web-console/frontend/app/components/form-field/components/form-field-size/controller.ts]: Can this be covered by tests?

Comment (Klaster1) [reply modules/web-console/frontend/app/components/form-field/components/form-field-size/controller.ts]: Yes.

File: modules/web-console/frontend/app/components/form-field/components/form-field-size/controller.ts [hunk 2]
```diff
@@ -76,7 +76,16 @@ export default class PCFormFieldSizeController<T> implements IInputErrorNotifier
...
+            const rawValue = this.ngModel.$viewValue;
+
+            if (rawValue) {
+                this._sizeScale = _.findLast(this.sizesMenu,
```
Comment (Klaster1) [inline modules/web-console/frontend/app/components/form-field/components/form-field-size/controller.ts]: Either import lodash or use native Array methods.
```suggestion
                this._sizeScale = [...this.sizesMenu].reverse().find(
```

File: modules/web-console/frontend/app/components/form-field/components/form-field-size/controller.ts (hunk 1 above)
Comment (vsisko) [reply modules/web-console/frontend/app/components/form-field/components/form-field-size/controller.ts]: Updated test "Switch editor measure" in modules/web-console/frontend/app/components/form-field/components/form-field-size/index.spec.ts to check initial measure detection.

File: modules/web-console/frontend/app/components/form-field/components/form-field-size/controller.ts (hunk 2 above)
Comment (vsisko) [reply modules/web-console/frontend/app/components/form-field/components/form-field-size/controller.ts]: Lodash has been imported.

**Comment by vsisko** [comment]
Covered by unit test in "Switch editor measure" in modules/web-console/frontend/app/components/form-field/components/form-field-size/index.spec.ts

**Pull request merged**